src/components/znkTimeline/timeline.service.js

```diff
@@ -25,11 +25,13 @@
             }
             if (ExerciseTypeEnum.SECTION) {
                 imgObj[ExerciseTypeEnum.SECTION.enum] = {icon: svgMap.section};
+                imgObj[ExerciseTypeEnum.SECTION.val] = {icon: svgMap.practice};
```

Comment:
I still not understand why we need to save both enum and val values in the map.
Let's discuss.